lib/sections/driver.js

```diff
@@ -1,60 +1,88 @@
 var utils = require(__dirname + '/utils.js');
 
-var driverJob = function(){
+var busJob = function(){
   return {
-    plugins: [],
-    pluginOptions: [],
     perform: function(args, callback){
       var self = this;
 
-      var started = 0;
-      var matches = [];
-      self.subscriptions(function(err, subscriptions, count){
-        if(count == 0){
-          if(started === 0 && typeof callback === 'function'){ callback(null, matches); }
-        }else{    
-          for(var app in subscriptions){
-            for(var i in subscriptions[app]){
-              started++;
-              var subscription = subscriptions[app][i];
-              var matched = subscriptionMatch(args, subscription);
-              if(matched === true){
-                matches.push(subscription['queue_name']);
-                var payload = driverMetadata(args, subscription.queue_name, app, i, subscription.class, args.bus_event_type);
-                self.queueObject.enqueue(subscription.queue_name, subscription.class, payload, function(err, toRun){
-                  started--;
-                  if(started === 0 && typeof callback === 'function'){ callback(null, matches); }
-                });
-              }else{
-                process.nextTick(function(){
-                  started--;
-                  if(started === 0 && typeof callback === 'function'){ callback(null, matches); }
-                });
-              }
-            }
+      if(args.bus_class_proxy === '::QueueBus::Driver'){   
+        driverPerform(self, args, callback); 
+      }else if(args.bus_class_proxy === '::QueueBus::Rider'){    
+        callback(new Error('TODO: ::QueueBus::Rider')); // TODO!
```

Comment:
Here's the code for rider.
https://github.com/queue-bus/queue-bus/blob/5c009a3b2d2d58994813bdca0dc78547a18b0295/lib/queue_bus/rider.rb

it basically looks up the function to execute from `bus_rider_sub_key` and `bus_rider_app_key`
You must have had this working in the current version, otherwise nothing would work. That is, it would never have received anything that was driven to it.